test/Feature/CBuffer/lit.local.cfg

```diff
@@ -1,4 +1,4 @@
-if 'Clang' in config.available_features:
+if 'Clang-Vulkan' in config.available_features:
     config.unsupported = True
```

Comment:
Should we just be enabling these now?

Reply:
Well, none of these tests pass under vulkan. I could remove this and add XFAILs to the tests directly if you think that makes more sense.

Reply:
I don't have a strong feeling, I just want to make sure that we don't somehow lose track and not enable the tests once they do start passing.